docs/upgrade-smart-agent.md

```diff
@@ -0,0 +1,146 @@
+# Overview
+
+On XX/XX/XXX, SignalFx will launch the Smart Agent, version 5.0.
+
+At a high-level, to better integrate with Splunk, this new version of the Smart Agent will be signed with Splunk certificates.
+
+SignalFx recommends that you upgrade your Smart Agent as soon as possible.
+
+Current users can continue to run Smart Agent 4.X; however, after XX/XX/XXX, any new installs or upgrades will require you to change to the new Splunk-signed repositories.
+
+## Upgrade the Smart Agent
+
+Review the following options to learn how to upgrade the Smart Agent:
+
+### Option 1: ATP
```

Comment:
The _Option #_ prefix doesn't really add much value here and they aren't really options, the user is pretty much bound to one of the three based on how they originally installed the agent.